subgraphs/rari-yag-v1/src/common/utils/constants.ts

```diff
@@ -0,0 +1,230 @@
+  "0x0000000000085d4780B73119b644AE5ecd22b376",
+); // TUSD
+TOKEN_MAPPING.set(
+  "0x54c512ac779647672b8d02e2fe2dc10f79bbf19f719d887221696215fd24e9f1",
+  "0x4Fabb145d64652a948d72533023f6E7A623C7C53",
+); //BUSD
+TOKEN_MAPPING.set(
+  "0x87ef9bf44f9ed3d4aeadafb38d9bc9470e7aac44fdcb9f7ffb957b862954cf2c",
+  "0x57Ab1ec28D129707052df4dF418D58a2D46d5f51",
+); // sUSD
+TOKEN_MAPPING.set(
+  "0x33d80a03b5585b94e68b56bdea4f57fd2e459401902cb2f61772e1b630afb4b2",
+  "0xe2f2a5C287993345a840Db3B0845fbC70f5935a5",
+); // mUSD
+
+//////////////////////////////////
+//// Rari Yield Pool Specific ////
+//////////////////////////////////
+
+export const YIELD_VAULT_ADDRESS = "0x9245efB59f6491Ed1652c2DD8a4880cBFADc3ffA"; // RariPoolController.sol
+export const YIELD_VAULT_MANAGER_ADDRESS = "0x59FA438cD0731EBF5F4cDCaf72D4960EFd13FCe6"; // RariPoolManager.sol
+export const RARI_YIELD_POOL_TOKEN = "0x3baa6B7Af0D72006d3ea770ca29100Eb848559ae";
+
+export const YIELD_VAULT_NAME = "Rari Yield Pool";
+export const YIELD_VAULT_SYMBOL = "RYPT"; // RYPT = rari yield pool token ie, R(X)PT
+
+/////////////////////////////////
+//// Rari USDC Pool Specific ////
+/////////////////////////////////
+
+export const USDC_VAULT_ADDRESS = "0x66f4856f1bbd1eb09e1c8d9d646f5a3a193da569"; // RariPoolController.sol
+export const USDC_VAULT_MANAGER_ADDRESS = "0xC6BF8C8A55f77686720E0a88e2Fd1fEEF58ddf4a"; // RariPoolManager.sol
+export const RARI_STABLE_POOL_TOKEN = "0x016bf078ABcaCB987f0589a6d3BEAdD4316922B0";
+
+export const USDC_VAULT_NAME = "Rari USDC Pool";
+export const USDC_VAULT_SYMBOL = "RSPT"; // RSPT = rari stable pool token
+
+////////////////////////////////
+//// Rari DAI Pool Specific ////
+////////////////////////////////
+
+export const DAI_VAULT_ADDRESS = "0xaFD2AaDE64E6Ea690173F6DE59Fc09F5C9190d74"; // RariPoolController.sol
+export const DAI_VAULT_MANAGER_ADDRESS = "0xB465BAF04C087Ce3ed1C266F96CA43f4847D9635"; // RariPoolManager.sol
+export const RARI_DAI_POOL_TOKEN = "0x0833cfcb11A5ba89FbAF73a407831c98aD2D7648";
+
+export const DAI_VAULT_NAME = "Rari DAI Pool";
+export const DAI_VAULT_SYMBOL = "RDPT"; // RDPT = rari dai pool token
+
+//////////////////////////////////
+//// Rari Ether Pool Specific ////
+//////////////////////////////////
+
+export const ETHER_VAULT_ADDRESS = "0x3F4931A8E9D4cdf8F56e7E8A8Cfe3BeDE0E43657"; // RariPoolController.sol
+export const ETHER_VAULT_MANAGER_ADDRESS = "0xD6e194aF3d9674b62D1b30Ec676030C23961275e"; // RariPoolManager.sol
+export const RARI_ETHER_POOL_TOKEN = "0xCda4770d65B4211364Cb870aD6bE19E7Ef1D65f4";
+
+export const ETHER_VAULT_NAME = "Rari Ether Pool";
+export const ETHER_VAULT_SYMBOL = "REPT"; // REPT = rariether pool token
+
+/////////////////////////////
+///// Protocol Specific /////
+/////////////////////////////
+
+export const RARI_DEPLOYER = "0xb8f02248d53f7edfa38e79263e743e9390f81942"; // using as "protocol address" b/c no factory contract
+export const PROTOCOL_NAME = "Earn";
```

Comment:
Is this what they call this product? If so, we can change it to "Rari Earn" and "rari-earn"

Reply:
On their website, it seems to be called Vaults. Maybe "Rari Vaults"?

Reply:
thoughts on:
`name` = "Rari Vaults v1"
`slug` = "rari-vaults-v1"

Reply:
on their ui looks like they label them "Vaults", but they are technically v1 vaults with v2 coming soon

Reply:
yeah, that sounds good!

Reply:
fixed